packages/renderer/src/lib/dialogs/QuickPickInput.spec.ts

```diff
@@ -208,6 +208,33 @@ describe('QuickPickInput', () => {
     expect(area).toBeInTheDocument();
   });
 
+  test.each([
+    { password: true, type: 'password' },
+    { password: false, type: 'text' },
+  ])('Expect that input is displayed correctly with type %s', async ({ password, type }) => {
+    const idRequest = 123;
+
+    const inputBoxOptions: InputBoxOptions = {
+      password: password,
+      validate: false,
+      placeHolder: 'Some placeholder',
+      prompt: 'Enter a value',
+      id: idRequest,
+    };
+
+    receiveFunctionMock.mockImplementation((message: string, callback: (options: InputBoxOptions) => void) => {
+      if (message === 'showInputBox:add') {
+        callback(inputBoxOptions);
+      }
+    });
+
+    render(QuickPickInput, {});
+
+    const area = await screen.findByPlaceholderText('Some placeholder');
+    expect(area).toBeInTheDocument();
+    expect(area).toHaveAttribute('type', type);
```

Comment:
```suggestion
    const { getByPlaceholderText } = render(QuickPickInput, {});

    const area = getByPlaceholderText('Some placeholder');
    expect(area).toBeInstanceOf(HTMLInputElement);
    expect(area).toHaveAttribute('type', type);
```

suggestion(non-blocking): personal preference
- to use result from render
- use `get` instead of `find`
- verify instance